cmd/gocharm/main.go

```diff
@@ -117,6 +118,9 @@ func main1(pkgPath string) error {
 	if err != nil {
 		return errgo.Notef(err, "cannot get current directory")
 	}
+	if *source && *noSource {
```

Comment:
All the other usage checking is directly in main, so I'd do this check there too, I think.